pkg/graph/resolvers/nda_agreement.go

```diff
@@ -0,0 +1,77 @@
+package resolvers
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/mint-app/pkg/authentication"
+	"github.com/cmsgov/mint-app/pkg/graph/model"
+	"github.com/cmsgov/mint-app/pkg/models"
+	"github.com/cmsgov/mint-app/pkg/storage"
+)
+
+//NDAAgreementGetByEUA returns an EUA agreement by eua
+func NDAAgreementGetByEUA(logger *zap.Logger, principal authentication.Principal, store *storage.Store) (*model.NDAInfo, error) {
+	nda, err := store.NDAAgreementGetByEUA(logger, principal.ID())
+	accepted := model.NDAInfo{}
```

Comment:
Perhaps we should call this variable something like `info`? `accepted` seems like it would be a boolean when reading.